grants_tagger/__main__.py

```diff
@@ -287,6 +292,19 @@ def wellcome_science(
     else:
         create_label_binarizer(output_path, label_binarizer_path)
 
+    if test_split:
+        data_dir, data_name = os.path.split(output_path)
+        train_output_path = os.path.join(data_dir, "train_" + data_name)
+        test_output_path = os.path.join(data_dir, "test_" + data_name)
```

Comment:
I actually don't like that idea anymore, I think it should be explicitly passed as a param. So one param per train, test and test can be empty if the user does not want to split

Reply:
@nsorros would you say in that case that we can get rid of output_path completely? and just have train_output_path and test_output_path?

Reply:
yep